### DISCUSSION TITLE
Does "clip-retrieval end2end" resume downloads if you run the same command again?

### DISCUSSION THREAD

**Discussion by ghost** [rom1504/clip-retrieval#183] [Q&A] [answered]
If you stop a run of `clip-retrieval end2end [input] [output]` and run the same command, will it continue from where it left off if the inputs and outputs are the same or will i start from scratch?

**Comment by ghost** [comment]
Answer seems to be yes from my testing.

**Comment by rom1504** [comment, marked as answer]
Img2dataset automatically resumes yes

The other steps can't resume
